apps/shared/avifpng.c

```diff
@@ -14,6 +14,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#if !defined(PNG_eXIf_SUPPORTED) || !defined(PNG_iTXt_SUPPORTED)
+#error "libpng 1.6.31 or above is required."
```

Comment:
I suppose those could be disabled in newer releases depending on the configuration, but that should be rare.